typed/declaration/examples.ipldsch

```diff
@@ -1,11 +1,8 @@
-
 type ExampleWithNullable map {String : nullable String}
 
 type ExampleWithAnonDefns struct {
 	fooField optional {String:String}
 	barField nullable {String:String}
 	bazField {String : nullable String}
 	wozField {String:[nullable String]}
-} representation map (
-	fooField: alias="foo_field"
```

Comment:
I gather aliases aren't something that landed in the latest schema, also the `:` and `=` characters on this line seem out of place, or was there some partly developed ideas here?